tutorials/connect-jamf-pro-to-smallstep.mdx

```diff
@@ -0,0 +1,195 @@
+    
+    For Privileges, choose:
+    
+    - Read Mobile Devices
+    - Read Computers
+    - Webhooks: Create, Delete, Read, Update
+4. Next, in the **API Clients** tab, add a **Client**
+    - Display name: Smallstep
+    - API Roles: Smallstep
+    - Enabled: Yes, choose Enable
+    - Choose **Save**
+5. Choose **Generate client secret**
+6. Temporarily save the **Client ID** and **Client Secret**. You’ll use them in the next step.
+
+### 2. Connect Jamf to Smallstep
+
+Let’s add the Jamf API credentials you just created to Smallstep.
+
+1. In the Smallstep UI, go to the [**Device Management**](https://smallstep.com/app/?next=/settings/devices) tab in ⛭ **Settings**
+2. Under Jamf, choose ➕ **Connect**
+3. Enter the API credentials from Jamf
+4. Choose **Add Platform.** Your device inventory will start syncing from Jamf to Smallstep.
+5. Temporarily save the **Webhook URL**, **Username**, and **Password** presented.
+
+### 3. Configure a SCEP Enrollment webhook in Jamf Pro
+
+Smallstep uses Dynamic SCEP certificate enrollment. This webhook allows Jamf to request a dynamic, short-lived challenge string from Smallstep during MDM enrollment. The challenge string is injected into a [SCEP payload](https://developer.apple.com/documentation/devicemanagement/scep) for your clients to authenticate to Smallstep.
+
+1. In Jamf Pro, choose ⚙️ **Settings**
+2. Under the **Global** tab, Choose **Webhooks**
+3. Add a new Webhook
+    1. Display name: Smallstep
+    2. Webhook URL: (details from step #2)
+    3. Authentication Type: Basic
+        1. Basic Authentication Username: (details from step #2)
+        2. Basic Authentication Password: (details from step #2)
+        3. Basic Authentication Verify Password: (details from step #2)
+    4. Content Type: JSON
+    5. Webhook Event: SCEPChallenge
+4. Choose **Save**
+
+### 4. Create Agent Package for Distribution
+
+In this step, you’ll upload the Smallstep agent package to Jamf’s distribution network.
+
+1. Download the latest package from [packages.smallstep.com](https://packages.smallstep.com/stable/darwin/step-agent-plugin_latest.pkg)
+2. In Jamf Pro, choose ⚙️ **Settings**
+3. Under the **Computer Management** tab, Choose **Packages**
+4. Add a new Package
+    1. Display name: Smallstep Agent
+    2. Filename: (upload from step #1)
+5. Choose **Save**
+
+### 5. Create Agent Script
+
+This step will install a script on your client devices that bootstraps the connection between your devices and Smallstep.
+
+1. In Jamf Pro, choose ⚙️ **Settings**
+2. Under the **Computer Management** tab, Choose **Scripts**
+3. Add a new Script
+    1. In the **General tab**, for **Display Name**: `Smallstep Agent Install`
+    2. In the Script tab:
+        - Mode: `Shell/Bash`
+        - Content:
+            
+            ```json
+            launchctl stop com.smallstep.launchd.Agent
+            /Applications/SmallstepAgent.app/Contents/MacOS/SmallstepAgent svc install
+            
+            ```
```

Comment:
Assuming syntax highlighting understands `console`; otherwise `shell` or `bash`?
```suggestion
            ```console
            launchctl stop com.smallstep.launchd.Agent
            /Applications/SmallstepAgent.app/Contents/MacOS/SmallstepAgent svc install
            ```
```